<launch>

  <node pkg="fwdis" name="teleop_joy" type="teleop_joy"/>

  <node pkg="joy" name="joy_node" type="joy_node">
    <param name="dev" value="/dev/input/js0"/>
  </node>

</launch>
